src/Application/SlackChannelChatterFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace MatchBot\Application;
+
+use Symfony\Component\Notifier\Bridge\Slack\SlackTransport;
+use Symfony\Component\Notifier\Chatter;
+use Symfony\Component\Notifier\ChatterInterface;
+
+class SlackChannelChatterFactory
```

Comment:
This class was partly motived by not knowing how to get PHP-DI to let us have two different services of the same type, but I don't think it's a bad class to have.